React/wbooks/src/BookDetail/index.js

```diff
@@ -1,37 +1,57 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import { BOOKS } from '../books_json';
+import ActionCreators from '../redux/books/actions';
 
 import BookDetailInfo from './Components/BookDetailInfo';
 import BookSuggestions from './Components/BookSuggestions';
 import CommentSection from './Components/CommentSection';
 
-function BookDetail({ match }) {
-  const bookId = match.params.id;
-  const book = BOOKS.find(b => b.id == bookId);
-  const suggestions = BOOKS.filter(b => b.genre == book.genre);
+class BookDetail extends Component {
+  componentDidMount() {
+    this.props.dispatch(ActionCreators.bookDetail(this.props.match.params.id));
+  }
+  render() {
+    const book = this.props.items ? this.props.items : null;
 
-  return (
-    <div>
-      <div className="body-container">
-        <Link to="/" className="back-button">
-          &lt; Volver
-        </Link>
-        <BookDetailInfo
-          imageUrl={book.image_url}
-          title={book.title}
-          author={book.author}
-          year={book.year}
-          genre={book.genre}
-        />
-        <hr />
-        <BookSuggestions suggestions={suggestions} />
-        <hr />
-        <CommentSection />
+    return (
+      <div>
+        <div className="body-container">
+          <Link to="/" className="back-button">
+            &lt; Volver
+          </Link>
+          {book && (
+            <div>
```

Comment:
you don't need the extra div, you didn't have it before

Reply:
You could use a `Fragment` here @axelfratoni 
```
import React, { Fragment } from 'react';
...
<Fragment>
  ...
</Fragment>
```